crates/bootstrap_srv/src/bin/kitsune2-bootstrap-srv.rs

```diff
@@ -0,0 +1,49 @@
+//! The binary kitsune2-bootstrap-srv.
+
+use kitsune2_bootstrap_srv::*;
+
+#[derive(clap::Parser, Debug)]
+#[command(version)]
+pub struct Args {
+    /// By default kitsune2-boot-srv runs in "testing" configuration
+    /// with much lighter resource usage settings. This testing mode
+    /// should be more than enough for most developer application testing
+    /// and continuous integration or automated tests.
+    ///
+    /// To setup the server to be ready to use most of the resources available
+    /// on a single given machine, you can set this "production" mode.
+    #[arg(long)]
+    pub production: bool,
+    // TODO - Implement the ability to specify TLS certificates
+    // TODO - Implement the ability to specify the listening address
+    // TODO - Implement the ability to override any other relevant
```

Comment:
Do we want to move these into issues in the kitsune2 repo?

Reply:
This PR is against the kitsune2 repo. Sorry, not sure what you mean here.

Reply:
[Done](https://github.com/holochain/kitsune2/issues)

Reply:
I'm going to decline this because if I change anything it'll dismiss the approval. But also because I don't really care about todos in the code. Feel free to make a PR to remove them! : )

Reply:
> But also because I don't really care about todos in the code.

Exactly, no one does! :grinning: